samples/ck-rx65n/basic-sample/src/iotconnect_app.c

```diff
@@ -266,6 +295,15 @@ bool app_startup(NX_IP *ip_ptr, NX_PACKET_POOL *pool_ptr, NX_DNS *dns_ptr) {
 	auth_driver_context = config->auth.data.x509.auth_interface_context;
 #endif // ENABLE_RX_TSIP_AUTH_DRIVER_SAMPLE
 #endif  // IOTCONNECT_SYMETRIC_KEY
+#endif  // IOTCONNECT_INTERACTIVE_SYMMETRIC_KEY
+
+    printf("config->cpid: '%s'\r\n", config->cpid);
+    printf("config->env: '%s'\r\n", config->env);
+    printf("config->duid: '%s'\r\n", config->duid);
+    printf("config->auth.type: %d\r\n", config->auth.type);
+    if(config->auth.type == IOTC_KEY) {
+        printf("config->auth.data.symmetric_key: '%s'\r\n", config->auth.data.symmetric_key);
```

Comment:
I've moved these printfs to within the INPUT_SYMMETRIC_KEY section.
If you typed it in then you know it / it acts as a double-check in case you mis-type.